browser-interface/packages/shared/web3.ts

```diff
@@ -280,13 +280,42 @@ const catalystABI = [
   }
 ]
 
+/**
+ * The HARDCODED_CATALYST_LIST exists because this function was taking ~8s fetching all the data from the contract (at
+ * least through metamask)
+ */
 export async function fetchCatalystNodesFromDAO(): Promise<CatalystNode[]> {
   if (!requestManager.provider) {
     throw new Error('requestManager.provider not set')
   }
 
   const net = await getAppNetwork()
 
+  if (HARDCODED_CATALYST_LIST) {
```

Comment:
I think this code is _never_ called, it is a fallback in the case peer.decentraland is down

Reply:
In my tests it was :/